actions/check-product-changes/poller.js

```diff
@@ -299,7 +299,7 @@ async function processDeletedProducts(remainingSkus, locale, state, counts, cont
   }
 }
 
-async function poll(params, aioLibs) {
+async function poll(params, aioLibs, skuFileName = 'check-product-changes/allSkus.json') {
```

Comment:
as the file name is hardcoded in get-all-skus, let's just make it a constant in utils or so